changelog.txt

```diff
@@ -20,6 +20,11 @@ Fixed
 
  - Fixed issues on Windows with ``H5Store``, project import/export, and operations that move files (#264, #266).
 
+Changed
++++++++
+
+ - Workspace directory is created with ``Project`` instance (#267, #271).
```

Comment:
I suggested a slight change in wording. Sometimes users will create signac projects from the command line, too. The command is `signac init PROJECT_NAME` so I think it's a bit clearer to say "initialize."
```suggestion
 - Workspace directory is created when ``Project`` is initialized (#267, #271).
```

Reply:
Could you move this "Changed" section above "Fixed" so that it matches the order of the other changelog sections? Typically I think we follow the order "Added, Changed, Fixed, Deprecated, Removed." (There may be a couple exceptions to this.)